app/src/main/java/com/portfolio/tasky/entry/EntryActivity.kt

```diff
@@ -67,7 +69,7 @@ class EntryActivity : AppCompatActivity(), FragmentInflater by FragmentInflaterI
         setTitle(getString(R.string.welcome_back))
 
         setFragmentManager(supportFragmentManager)
-        val loginFragment = LoginFragment.getInstance()
+        val loginFragment = RegistrationFragment.getInstance()
```

Comment:
login fragment yet you get an instance of a RegistrationFragment?

Reply:
I just switch the fragment here for convenience.

Done.